From: 13832136087 <ningxiyuyi@126.com>
Date: Sun, 30 Aug 2026 13:35:38 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (33 of 33 strings)

Translation: openkylin-huanghe/ukui-power-manager-tray
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-power-manager-tray/ko_KP/
---
 .../translations/ukui-power-manager-tray_ko_KP.ts        | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/ukui-power-manager-tray/translations/ukui-power-manager-tray_ko_KP.ts b/ukui-power-manager-tray/translations/ukui-power-manager-tray_ko_KP.ts
index 145a1a6..ef65ff8 100644
--- a/ukui-power-manager-tray/translations/ukui-power-manager-tray_ko_KP.ts
+++ b/ukui-power-manager-tray/translations/ukui-power-manager-tray_ko_KP.ts
@@ -12,7 +12,7 @@
         <location filename="../powertray.cpp" line="155"/>
         <location filename="../powertray.cpp" line="178"/>
         <source>No battery detected</source>
-        <translation>배터리를 감지하지 못했습니다</translation>
+        <translation>배터리가 감지되지 않았습니다</translation>
     </message>
     <message>
         <location filename="../powertray.cpp" line="163"/>
@@ -26,7 +26,7 @@
         <location filename="../powertray.cpp" line="203"/>
         <location filename="../powertray.cpp" line="212"/>
         <source>%1% available (plugged in)</source>
-        <translation>%1% 가용 (전원 련결됨)</translation>
+        <translation>%1% 사용 가능 (전원 련결됨)</translation>
     </message>
     <message>
         <location filename="../powertray.cpp" line="168"/>
@@ -35,17 +35,17 @@
         <location filename="../powertray.cpp" line="233"/>
         <location filename="../powertray.cpp" line="244"/>
         <source>%1% remaining</source>
-        <translation>%1% 가용</translation>
+        <translation>%1% 사용 가능</translation>
     </message>
     <message>
         <location filename="../powertray.cpp" line="208"/>
         <source>%1 min to fully charge (%2%)</source>
-        <translation>남은 %1 분 후 충전 완료 (%2%)</translation>
+        <translation>%1 분 후 충전 완료 (%2%)</translation>
     </message>
     <message>
         <location filename="../powertray.cpp" line="214"/>
         <source>%1 hr %2 min to fully charge (%3%)</source>
-        <translation>남은 %1 시간 %2 분 후 충전 완료 (%3%)</translation>
+        <translation>%1 시간 %2 분 후 충전 완료 (%3%)</translation>
     </message>
     <message>
         <location filename="../powertray.cpp" line="229"/>
@@ -85,12 +85,12 @@
     <message>
         <location filename="../powerwindow.cpp" line="177"/>
         <source>Endurance</source>
-        <translation>최고의 에너지 효율</translation>
+        <translation>최적 에너지 효율</translation>
     </message>
     <message>
         <location filename="../powerwindow.cpp" line="185"/>
         <source>Performance</source>
-        <translation>최고의 성능</translation>
+        <translation>최적 성능</translation>
     </message>
     <message>
         <location filename="../powerwindow.cpp" line="189"/>
@@ -112,7 +112,7 @@
         <location filename="../powerwindow.cpp" line="302"/>
         <location filename="../powerwindow.cpp" line="352"/>
         <source>No Battery detected</source>
-        <translation>배터리를 감지하지 못했습니다</translation>
+        <translation>배터리가 감지되지 않았습니다</translation>
     </message>
     <message>
         <location filename="../powerwindow.cpp" line="383"/>
